### PR TITLE
publishTranscription

### PR DESCRIPTION
**Pull request by bcherry**

### CODE REVIEW

File: .changeset/nasty-ravens-live.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@livekit/rtc-node": patch
```
Comment (nbsp) [inline .changeset/nasty-ravens-live.md:2]:
```suggestion
"@livekit/rtc-node": minor
```

? cc @theomonnom your call

File: packages/livekit-rtc/src/participant.ts [hunk 2]
```diff
@@ -130,6 +135,34 @@ export class LocalParticipant extends Participant {
     }
   }
 
+  async publishTranscription(transcription: Transcription): Promise<void> {
```
Comment (nbsp) [inline packages/livekit-rtc/src/participant.ts:138]: nit: no need for `: Promise<void>`

File: packages/livekit-rtc/src/transcription.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+export interface TranscriptionSegment {
+  id: string;
+  text: string;
+  startTime: number;
+  endTime: number;
```
Comment (nbsp) [inline packages/livekit-rtc/src/transcription.ts:5]: should these be BigInts here to begin with?

**Review by nbsp — COMMENTED** [review]
\+ reuse header on transcription.ts

File: packages/livekit-rtc/src/transcription.ts (hunk 3 above)
Comment (bcherry) [reply packages/livekit-rtc/src/transcription.ts]: yeah that makes sense. of course they're all 0 everywhere anyways!

File: .changeset/nasty-ravens-live.md (hunk 1 above)
Comment (bcherry) [reply .changeset/nasty-ravens-live.md]: yeah you're probably right. i changed that but am happy to change back

**Review by nbsp — APPROVED** [review]
looks good, just run `format:write` to fix tests

**Pull request merged**